catacomb/catacomb.go

```diff
@@ -221,6 +225,12 @@ func (catacomb *Catacomb) Context(parent context.Context) context.Context {
 	return catacomb.tomb.Context(parent)
 }
 
+// scopedContext returns a context that is associated with the catacomb's
+// tomb.Context, and will be cancelled when the tomb dies.
+func (catacomb *Catacomb) scopedContext() context.Context {
```

Comment:
According to the doc string of Context: https://pkg.go.dev/github.com/juju/worker/v3/catacomb#Catacomb.Context

This is just `catacomb.Context(nil)`. Is it worth the extra function (it is if it makes it much clearer what you're trying to express, I'm just trying to ask the question.)

Reply:
I'd rather be explicit. If this does change to something other than `context.Background()` and that isn't the default we prefer, then it's a pain to change.